course_info/static/course_info/js/controllers/SearchController.js

```diff
@@ -215,7 +215,8 @@
                     dom: '<<t>ip>',
                     language: {
                         info: 'Showing _START_ to _END_ of _TOTAL_ courses',
-                        emptyTable: 'There are no courses to display.',
+                        emptyTable: 'Your search didn’t generate any results.' +
```

Comment:
Any reason this is a `&rsquo;` instead of an `&apos;`? It probably should be a regular apostrophe (and simply escaped or wrapped in a double-quote string declaration).